app/views/episode_metrics/_overview.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="container">
+  <div class="my-4">
+    <div class="col-12 d-flex justify-content-between">
+      <div>
+        <h1 class="fw-bold lh-1"><%= episode.title %></h1>
+        <div class="d-flex gap-2 align-items-center">
+          <p><%= local_date(episode.first_rss_published_at, format: :full_month) %></p>
+          <p>&bull;</p>
+          <p><%= episode.readable_episode_duration %></p>
+        </div>
+      </div>
+      <div>
+        <%= link_to episode.enclosure_url, class: "btn btn-sm btn-success" do %>
+          <%= t(".audio_link") %>
+          <span class="material-icons">link</span>
+        <% end %>
+        <%= link_to episode.url, class: "btn btn-sm btn-success" do %>
+          <%= t(".episode_url") %>
+          <span class="material-icons">open_in_new</span>
+        <% end %>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-8 mb-4">
+        <div>
+          <%= embed_player_episode_iframe(episode) %>
+        </div>
+        <div class="lh-sm"><%= sanitize(episode.description) %></div>
+      </div>
+      <div class="col-4 mb-4">
+        <div class="card shadow border-0">
+          <div class="card-header card-header-light">
+            <%= t(".card_header") %>
+          </div>
+          <div class="card-body">
+            <%= t(".card_body") %>
+          </div>
+          <div class="card-footer p-0">
+            <ul class="list-group list-group-flush">
+              <li class="list-group-item"><%= t(".rss") %></li>
+              <li class="list-group-item"><%= t(".apple") %></li>
+              <li class="list-group-item"><%= t(".spotify") %></li>
```

Comment:
not entirely sure where to pull the statuses from for this list. the particular feed?